hack/update-codegen.sh

```diff
@@ -0,0 +1,187 @@
+#! /usr/bin/env bash
```

Comment:
My initial idea was to let knobot execute the script during auto-update. Because it's well know and executed per those scripts. But we can change name now, I'm pretty sure GH actions are OK way to go for this kind of updates.

Reply:
If needed we can always reintroduce a wrapper script that `update-codegen.sh` that calls out to this script.